connectors/src/api/webhooks/webhook_firecrawl.ts

```diff
@@ -32,52 +40,163 @@ const _webhookFirecrawlAPIHandler = async (
         };
       }>;
       metadata: {
-        [key: string]: unknown;
+        connectorId: number;
```

Comment:
metadata is a `Record<string, string>` on the input side, so parsing  necessary
```suggestion
        connectorId: string;
```

Reply:
:+1: